src/ck3/data/messages.rs

```diff
@@ -62,8 +62,32 @@ impl DbKind for MessageFilterType {
         let loca = format!("message_filter_{key}");
         data.verify_exists_implied(Item::Localization, &loca, key);
 
-        vd.field_choice("display", &["feed", "toast", "hidden"]);
+        vd.field_choice("display", &["feed", "toast", "hidden", "popup"]);
         vd.field_bool("always_show");
         vd.field_bool("auto_pause");
+        vd.field_integer("sort_order");
+        vd.field_item("group", Item::MessageFilterGroupType);
```

Comment:
Where did you get these two fields? I don't see them in the docs or in the game scripts

Reply:
Removed from file